build/shared/lib/languages/PDE.properties

```diff
@@ -445,6 +445,18 @@ ensure_exist.messages.unrecoverable.description = Could not properly re-save the
 # Check name
 check_name.messages.is_name_modified = The sketch name had to be modified. Sketch names can only consist\nof ASCII characters and numbers (but cannot start with a number).\nThey should also be less than 64 characters long.
 
+# External changes detector
+change_detect.reload.title=Tab modified externally
+change_detect.reload.question="%s" was modified by another program.
+change_detect.reload.comment=Would you like to keep this version or load the new changes?\nEither way, the version you discard will be saved to your sketch folder.
```

Comment:
Ok, understood. I think my problem was with 'keep this version'. What is a version? Does *this* refer to the externally modified version from the previous sentence?

Something like 'Do you want to keep what is in the *test5* tab or reload the tab from the file?' may be better?

Just a proposal, it's your call.